chronos/asyncfutures2.nim

```diff
@@ -243,6 +243,12 @@ template complete*[T](future: Future[T], val: T) =
   ## Completes ``future`` with value ``val``.
   complete(future, val, getSrcLocation())
 
+template complete*[T](F: type Future[T], val: T, fromProc: static[string] = ""): F =
```

Comment:
I see this named confusing, and this could introduce some sandwich problem in the future, i think such templates should have different name, like `completed` for example.